src/Arcus.BackgroundJobs.Tests.Unit/Databricks/DatabricksInfoProviderTests.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Arcus.BackgroundJobs.Databricks;
+using Bogus;
+using Microsoft.Azure.Databricks.Client;
+using Microsoft.Extensions.Logging.Abstractions;
+using Xunit;
+
+namespace Arcus.BackgroundJobs.Tests.Unit.Databricks
+{
+    [Trait(name: "Category", value: "Unit")]
+    public class DatabricksInfoProviderTests
```

Comment:
Maybe provide tests for the other methods as well?

Reply:
Yes, indeed. Were missing.